appserver/extras/embedded/tests/embedded-vs-jersey/pom.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
Moved, because this tests were too far from building extras and payara embedded, which is hidden under a profile. That had two consequences:
- the Payara Samples tested against old version of Payara Embedded 
- sometimes the embedded wasn't built yet -> dependency issues.